### PR TITLE
feat: Associate apps to projects

### PR DESCRIPTION
**Pull request by JigarJoshi**
Adding optional parameter that enables platform to associate project <> applications.

### CODE REVIEW

**Review by rbarabas — DISMISSED** [review]

File: server/v1/management.proto [hunk 1]
```diff
@@ -73,6 +73,8 @@ message CreateApplicationRequest {
   string name = 1;
   // A human readable app description
   string description = 2;
+  // Project(s) name
+  repeated string projects = 3;
```
Comment (efirs) [inline server/v1/management.proto:77]: I think application key should belong to only one project.
```
Project
  AppKeys
```

Comment (JigarJoshi) [reply server/v1/management.proto:77]: This is to support existing app keys to multiple projects for backward compatibility.

Comment (JigarJoshi) [reply server/v1/management.proto]: During internal migration we will take care of this - you are right we can avoid exposing this to user interface. Made an update.

**Review by himank — APPROVED** [review]

File: server/v1/management.proto [hunk 2]
```diff
@@ -115,10 +117,14 @@ message Application {
   int64 updated_at = 7;
   // Updated by
   string updated_by = 8;
+  // Project(s) it belongs to (for backward compatibility - multiple projects)
+  repeated string projects = 9;
```
Comment (efirs) [inline server/v1/management.proto:121]: Here too.

**Pull request merged**